### PR TITLE
Added Sample

### PR DESCRIPTION
**Pull request by jamescrosswell**

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<Project>
+    <PropertyGroup>
+        <SentryOrg>sentry-sdks</SentryOrg>
```
Comment (jamescrosswell) [inline Directory.Build.props]: We should probably remove these...

File: AspireWithSentry.ServiceDefaults/Extensions.cs [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+        return builder;
+    }
+
+    public static WebApplicationBuilder AddSentry(this WebApplicationBuilder builder, Action<SentryAspNetCoreOptions>? configureOptions = null)
+    {
+        builder.WebHost.UseSentry(options =>
+        {
+            // Change this to the DSN of your Sentry project
+            // options.Dsn = "... your DSN here ...";  // Replace this!!!
```
Comment (bruno-garcia) [inline AspireWithSentry.ServiceDefaults/Extensions.cs:42]: if we uncomment, and assign it to an invalid value, will it make it easier for folks to find out what to change?

`= "create a project in sentry.io to get this value";`

File: Directory.Build.props (hunk 1 above)
Comment (bruno-garcia) [reply Directory.Build.props]: yeah prob good to add also some comments about where to get it. Maybe link to docs

File: AspireWithSentry.ServiceDefaults/Extensions.cs (hunk 2 above)
Comment (jamescrosswell) [reply AspireWithSentry.ServiceDefaults/Extensions.cs:42]: I've added an explicit exception that will throw until the instructions from the README.md have been completed.

**Review by bitsandfoxes — APPROVED** [review]

File: AspireWithSentry.ApiService/Program.cs [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        .ToArray();
+    return forecast;
+});
+
+app.MapGet("/crashtest", Delegate(SqlConnection connection) =>
+{
+    using (connection)
+    {
+        connection.Open();
+        var command = new SqlCommand(CommandHelper.CommandText, connection);
```
Comment (bruno-garcia) [inline AspireWithSentry.ApiService/Program.cs:46]: `using`?

File: AspireWithSentry.ServiceDefaults/Extensions.cs (hunk 2 above)
Comment (bruno-garcia) [reply AspireWithSentry.ServiceDefaults/Extensions.cs:42]: That's a pretty cool solution!

**Review by bruno-garcia — APPROVED** [review]

**Pull request merged**